backend/src/utils/mailer.ts

```diff
@@ -0,0 +1,23 @@
+import nodemailer from 'nodemailer';
+
+async function sendEmail(to: string, subject: string, text: string) {
+  // Create a transporter object using Gmail SMTP transport
+  let transporter = nodemailer.createTransport({
+    service: 'gmail',
+    auth: {
+      user: 'fraud.ninja.team@gmail.com', // Replace with your Gmail address
+      pass: 'nosq zgue eloq mzaj', // Replace with your App Password or Gmail password
```

Comment:
Should `pass` be read from `.env`? Also I'm assuming these comments should be removed